graphite/client.go

```diff
@@ -62,6 +62,7 @@ func NewClient(carbon string, carbon_transport string, write_timeout time.Durati
 		fileConf, err = config.LoadFile(configFile)
 		if err != nil {
 			log.With("err", err).Warnln("Error loading config file")
+			fileConf = &config.Config{}
```

Comment:
there is two options:
- we either re-set the default config even when there is an error in LoadFile (fileConf == nil)
- or we return and start without reader/writer (we also might quit)